RealmModelGenerator/NibDesignable.swift

```diff
@@ -72,7 +72,7 @@ extension NSView {
         return String(type(of: self).description().split(separator: ".").last!)
     }
     
-    public func nibDidLoad() {}
+    // public func nibDidLoad() {}
```

Comment:
remove commented out method. It's added below

Reply:
Done